hack/prom-rule-ci/prom-rules-tests.yaml

```diff
@@ -769,3 +769,69 @@ tests:
       - eval_time: 30h
         alertname: KubeVirtVMIExcessiveMigrations
         exp_alerts: []
+
+  # No nodes are available to host VMs
+  - interval: 1m
```

Comment:
As it is difficult to test (e2e) scenario "schedulable" present and kvm not present I suggest testing it here.

Reply:
already there, but added a description to each test

Reply:
Sorry, I think we are missing a case: Having schedulable, missing kvm but emulation enabled.

Reply:
right, and also `no schedulable, having kvm but emulation enabled`
but we are already testing `without emulation` that both `has kvm allocatable but no schedulable label` and `has schedulable label but no kvm allocatable` fire the alert

so it should be enough to test `no kvm allocatable and no schedulable, but has emulation enabled` fires no alert to know that it won't fire the alert in the other "less strong cases", do you think is necessary to add the test for each specific configuration?

Reply:
I beg to oppose. The most relevant tests are:

Having schedulable and
-     no KVM but emulation
-     KVM with/out emulation

This test is virtually free so I don't see why we should be reluctant to add it.